recipes/arrow/all/conanfile.py

```diff
@@ -514,6 +516,8 @@ def package_info(self):
             self.cpp_info.components["dataset"].libs = ["arrow_dataset"]
             if self.options.parquet:
                 self.cpp_info.components["dataset"].requires = ["libparquet"]
+            if self.options.acero and Version(self.version) >= "19.0.0":
+                self.cpp_info.components["dataset"].requires = ["libacero"]
```

Comment:
Good catch! I've double checked that this can be seen in the diff/generated pkgconf files when compiling the library